lib/errors.js

```diff
@@ -63,18 +63,13 @@ function toGraphQLError (err) {
 
 function defaultErrorFormatter (err, ctx) {
   let errors = [{ message: err.message }]
-  let log
-
-  if (ctx) {
```

Comment:
why is this condition removed? We're seeing

```
TypeError: Cannot read properties of undefined (reading 'reply')
    at defaultErrorFormatter (/app/node_modules/mercurius/lib/errors.js:67:19)
    at Object.<anonymous> (/app/node_modules/mercurius/lib/routes.js:139:40)
    at handleError (/app/node_modules/fastify/lib/reply.js:555:20)
    at onErrorHook (/app/node_modules/fastify/lib/reply.js:524:5)
    at _Reply.Reply.send (/app/node_modules/fastify/lib/reply.js:128:5)
    at done (/app/node_modules/fastify/lib/contentTypeParser.js:167:13)
    at Parser.defaultJsonParser [as fn] (/app/node_modules/fastify/lib/contentTypeParser.js:262:14)
    at IncomingMessage.onEnd (/app/node_modules/fastify/lib/contentTypeParser.js:243:27)
    at AsyncResource.runInAsyncScope (node:async_hooks:199:9)
    at IncomingMessage.emit (node:events:402:35)
```

in production after updating

Reply:
That's not supposed to happen due other constraints in this PR. That's definitely a bug but we will need a repro for it.

Reply:
I think I've found the problem, good spot. I'll open an issue